Add v9.0.0 changelog entry

rekmarks opened the pull request:
The v9 changelog update. Contains some extra formatting changes to the file because I decided to run `prettier` on it to get rid of syntax warnings in my editor.

rekmarks committed:
Add 9.0.0 changelog

rekmarks committed:
Run prettier on the changelog [skip e2e]

rekmarks committed:
Restore current develop branch header [skip e2e]

Gudahtt commented on CHANGELOG.md:
```diff
@@ -2,16 +2,37 @@
+- [#10000](https://github.com/MetaMask/metamask-extension/pull/10000): Use consistent font size for modal top right Close links
+- [#10046](https://github.com/MetaMask/metamask-extension/pull/10046): Improve home screen notification appearance
+- [#10117](https://github.com/MetaMask/metamask-extension/pull/10117): Fix network settings Kovan block explorer link
+- [#10040](https://github.com/MetaMask/metamask-extension/pull/10040): Disable console in contentscript to reduce noise
```
This commit was included in v8.1.9; I'll move it there.

Gudahtt commented on CHANGELOG.md:
```diff
@@ -2,16 +2,37 @@
+- [#10046](https://github.com/MetaMask/metamask-extension/pull/10046): Improve home screen notification appearance
+- [#10117](https://github.com/MetaMask/metamask-extension/pull/10117): Fix network settings Kovan block explorer link
+- [#10040](https://github.com/MetaMask/metamask-extension/pull/10040): Disable console in contentscript to reduce noise
+- [#10070](https://github.com/MetaMask/metamask-extension/pull/10070): Enable eth_getProof RPC method
```
This was included in v8.1.9 already, so I will remove it.

Gudahtt commented on CHANGELOG.md:
```diff
@@ -2,16 +2,37 @@
+- [#10100](https://github.com/MetaMask/metamask-extension/pull/10100): Remove unnecessary swaps footer space when in dropdown mode
+- [#10069](https://github.com/MetaMask/metamask-extension/pull/10069): Fetch swap quote refresh time from API
+- [#10045](https://github.com/MetaMask/metamask-extension/pull/10045): Fix token validation in Send flow
+- [#10048](https://github.com/MetaMask/metamask-extension/pull/10048): Fix display of boolean values when signing typed data
```
This was included in v8.1.9, so I'll remove it here.

Gudahtt commented on CHANGELOG.md:
```diff
@@ -2,16 +2,37 @@
+- [#10135](https://github.com/MetaMask/metamask-extension/pull/10135): Fix error where a swap only completed the token approval transaction
+- [#10100](https://github.com/MetaMask/metamask-extension/pull/10100): Remove unnecessary swaps footer space when in dropdown mode
+- [#10069](https://github.com/MetaMask/metamask-extension/pull/10069): Fetch swap quote refresh time from API
+- [#10045](https://github.com/MetaMask/metamask-extension/pull/10045): Fix token validation in Send flow
```
This was included in v8.1.9, so I'll remove it here.

Gudahtt commented on CHANGELOG.md:
```diff
@@ -2,16 +2,37 @@
+## 9.0.0 Fri Jan 8 2021
+
+- [#9156](https://github.com/MetaMask/metamask-extension/pull/9156): Remove window.web3 injection
+- [#10039](https://github.com/MetaMask/metamask-extension/pull/10039): Add web3 shim usage notification (#10039)
```
Nit: the PR number here is redundant

Gudahtt committed:
Remove duplicate entries, and move #10040 entry to v8.1.9

Any entries that were duplicated or in the wrong release have been
removed/moved.

Gudahtt committed:
Update changelog to include both new v9 changes

Gudahtt reviewed (commented):
I've just updated this to include the two most recent commits, and I've addressed some fairly straightforward feedback from my review.

Gudahtt reviewed (approved):
This all looks good to me now! Please take a look at my changes before you merge this @rekmarks to make sure they all look OK

rekmarks committed:
Remove more duplicate changelog entries [skip e2e]
